packages/nuka/src/slider-list.tsx

```diff
@@ -125,6 +124,23 @@ export const SliderList = React.forwardRef<HTMLDivElement, SliderListProps>(
       ...percentOffsetForSlideProps
     );
 
+    const isEdgeEasing =
+      !disableEdgeSwiping &&
+      !wrapAround &&
+      ((currentSlide === dotIndexes[0] && animationDistance < 0) ||
+        (currentSlide === dotIndexes[dotIndexes.length - 1] &&
+          animationDistance > 0));
+    const { value: transition, isAnimating } = useTween(
+      speed,
+      !isEdgeEasing ? easing : edgeEasing,
+      // animationDistance is assumed to be unique enough that it can be used to
+      // detect when a new animation should start. This is used in addition to
+      // currentSlide because some animations, such as those with edgeEasing, do
+      // not occur due to a change in value of currentSlide
+      currentSlide + animationDistance,
```

Comment:
If I were to be a bit less hacky/ more verbose with the extra dependency on animationDistance, it would have doubled the code in some areas (see the ref storing and comparison in `useTween`), and the logic for each of the two dependencies would have been identical.